content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md

```diff
@@ -16,52 +16,52 @@ GitHub Desktop keyboard shortcuts on macOS
-|<kbd>⌘</kbd><kbd>H</kbd> | Hide the {% data variables.product.prodname_desktop %} application
-|<kbd>⌥</kbd><kbd>⌘</kbd><kbd>H</kbd> | Hide all other applications
-|<kbd>⌘</kbd><kbd>Q</kbd> | Quit {% data variables.product.prodname_desktop %}
-|<kbd>⌃</kbd><kbd>⌘</kbd><kbd>F</kbd> | Toggle full screen view
-|<kbd>⌘</kbd><kbd>0</kbd> | Reset zoom to default text size
-|<kbd>⌘</kbd><kbd>=</kbd> | Zoom in for larger text and graphics
-|<kbd>⌘</kbd><kbd>-</kbd> | Zoom out for smaller text and graphics
-|<kbd>⌥</kbd><kbd>⌘</kbd><kbd>I</kbd> | Toggle Developer Tools
+|<kbd>Command</kbd>+<kbd>,</kbd> | Go to Preferences
+|<kbd>Command</kbd>+<kbd>H</kbd> | Hide the {% data variables.product.prodname_desktop %} application
+|<kbd>Option</kbd>+<kbd>Command</kbd>+<kbd>H</kbd> | Hide all other applications
+|<kbd>Command</kbd>+<kbd>Q</kbd> | Quit {% data variables.product.prodname_desktop %}
+|<kbd>Control</kbd>+<kbd>Command</kbd>+<kbd>F</kbd> | Toggle full screen view
+|<kbd>Command</kbd>+<kbd>0</kbd> | Reset zoom to default text size
+|<kbd>Command</kbd>+<kbd>=</kbd> | Zoom in for larger text and graphics
+|<kbd>Command</kbd>+<kbd>-</kbd> | Zoom out for smaller text and graphics
+|<kbd>Option</kbd>+<kbd>Command</kbd>+<kbd>I</kbd> | Toggle Developer Tools
 
 ## Repositories
 
 | Keyboard shortcut | Description
 |-----------|------------
-|<kbd>⌘</kbd><kbd>N</kbd> | Add a new repository
-|<kbd>⌘</kbd><kbd>O</kbd> | Add a local repository
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>O</kbd> | Clone a repository from {% data variables.product.prodname_dotcom %}
-|<kbd>⌘</kbd><kbd>T</kbd> | Show a list of your repositories
-|<kbd>⌘</kbd><kbd>P</kbd> | Push the latest commits to {% data variables.product.prodname_dotcom %}
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>P</kbd> | Pull down the latest changes from {% data variables.product.prodname_dotcom %}
-|<kbd>⌘</kbd><kbd>⌫</kbd> | Remove an existing repository
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>G</kbd> | View the repository on {% data variables.product.prodname_dotcom %}
-|<kbd>⌃</kbd><kbd>&grave;</kbd> | Open repository in your preferred terminal tool
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>F</kbd> | Show the repository in Finder
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>A</kbd> | Open the repository in your preferred editor tool
-|<kbd>⌘</kbd><kbd>I</kbd> | Create an issue on {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>N</kbd> | Add a new repository
+|<kbd>Command</kbd>+<kbd>O</kbd> | Add a local repository
+|<kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>O</kbd> | Clone a repository from {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>T</kbd> | Show a list of your repositories
+|<kbd>Command</kbd>+<kbd>P</kbd> | Push the latest commits to {% data variables.product.prodname_dotcom %}
+|<kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>P</kbd> | Pull down the latest changes from {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>⌫</kbd> | Remove an existing repository
```

Comment:
is the icon ok here or should it be `delete`/`backspace`

Reply:
@AnilSeervi It should be `Delete` as it is spelled as delete on Mac keyboard.

Reply:
Changed it to `Delete`